### DISCUSSION TITLE
Multiple DBs for application communication?

### DISCUSSION THREAD

**Discussion by arvanasse** [sidekiq/sidekiq#5280] [Q&A] [answered]
I have two copies of the same application running on separate servers in order to support 'sandbox' and 'production' environments for our customers.  Each uses a unique redis DB within the same redis instance for it's background processing (ActiveJob with Sidekiq::Worker included).  So far, they all play nicely together.

I would like to create a 'transfer' process to move content from the 'sandbox' environment to the 'production' environment for the moment when customers upgrade to a paid plan.  Is it possible to configure a Sidekiq::Worker in the 'sandbox' environment to enqueue a job in the redis db for the 'production' instance (and conversely the same worker in the 'production' instance to execute the job)?  The queueing would only from from sandbox => production.

**Comment by mperham** [comment, marked as answer]
Yes, this is called sharding. There is a wiki page on it.

**Reply by arvanasse** [reply]
Thanks, Mike.  I thought the solution was slightly different than the sharding example but I'll dig in on the details a bit more closely.
